ostree/gen-fixtures.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+DIR=""
+TREE=$(mktemp -d /tmp/tree.XXXXX)
+USAGE="
+Usage: gen-fixtures.sh <output-dir>
+
+Generate an OSTree repository with (2) branches:
+ - rawhide
+ - stable
+
+Each branch will have (3) commits for versions:
+ - 1.0
+ - 1.1
+ - 1.2
+
+The OSTree repository will be created in <output-dir>.
+The directory will be be created.
+"
+
+show_help()
+{
+  echo "${USAGE}"
+}
+
+_mkdir()
+{
+  if [ -e ${DIR} ]; then
+    if [ ! -z "$(ls -A ${DIR})" ]; then
+      echo 1>&2 ""
+      echo 1>&2 "${DIR} must be empty."
+      echo 1>&2 ""
+      exit 1
+    fi
+  else
+    mkdir -p "${DIR}"
+  fi
+}
+
+gen_tree()
+{
+  cp "$0" "${TREE}"
+}
+
+gen_repository()
+{
+  ostree init --repo=${DIR} --mode=archive-z2
+  # branch: rawhide
+  ostree commit --repo=${DIR} -b rawhide --add-metadata-string=version=1.1 "${TREE}"
+  ostree commit --repo=${DIR} -b rawhide --add-metadata-string=version=1.2 "${TREE}"
+  ostree commit --repo=${DIR} -b rawhide --add-metadata-string=version=1.3 "${TREE}"
+  # branch: stable
+  ostree commit --repo=${DIR} -b stable --add-metadata-string=version=1.1 "${TREE}"
+  ostree commit --repo=${DIR} -b stable --add-metadata-string=version=1.2 "${TREE}"
+  ostree commit --repo=${DIR} -b stable --add-metadata-string=version=1.3 "${TREE}"
+  # summary
+  ostree summary -u --repo="${DIR}"
+}
+
+clean()
+{
+  rm -rf "${TREE}"
+}
+
+
+if [ "$#" -ne "1" ]; then
+  show_help
+  exit 1
+fi
+
+DIR=$1
```

Comment:
Please quote this. This will break if the script is called as `ostree/gen-fixtures.sh "fixtures/ostree/small tree"`.